Sources/ComposableArchitecture/UIKit/IfLetUIKit.swift

```diff
@@ -75,16 +75,4 @@ extension Store {
       unwrapCancellable.cancel()
     }
   }
-
-  /// An overload of `ifLet(then:else:)` for the times that you do not want to handle the `else`
```

Comment:
This overload was originally added to support trailing closure syntax (`.ifLet { ... }`), but since Swift 5.3, we get this automatically because of [SE-0279](https://github.com/apple/swift-evolution/blob/80602141f65b3fc8a98283e1e8dc4fe1b51312bd/proposals/0279-multiple-trailing-closures.md).

(This'll probably break CI, but just wanted to discuss. Can also bring this back.)